charts/playbooks-flux/templates/request-namespace-access.yaml

```diff
@@ -43,6 +42,7 @@ spec:
         filter:
           - types:
               - Kubernetes::ClusterRole
+              - Kubernetes::Role
```

Comment:
I don't think this would would - it would need to be a role in the namespace we are granting access to

Reply:
Oh, I see